src/main/java/com/itschool/eventmanagment/models/dtos/ParticipantDTO.java

```diff
@@ -1,16 +1,40 @@
 package com.itschool.eventmanagment.models.dtos;
 
+import com.itschool.eventmanagment.models.entities.Participant;
 import lombok.Data;
 
 @Data
 public class ParticipantDTO {
+    private Long id;
     private String firstName;
     private String lastName;
     private String email;
 
+    ParticipantDTO() {
+
+    }
+
     public ParticipantDTO(String firstName, String lastName, String email) {
         this.firstName = firstName;
         this.lastName = lastName;
         this.email = email;
     }
+
+    public ParticipantDTO(Long id, String firstName, String lastName, String email) {
+        this.id = id;
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.email = email;
+    }
+
+
+
+    public static ParticipantDTO fromParticipant(Participant participant) {
```

Comment:
Unusual static methods here. I have not evaluated yet where this is used, but DTOs should be similar to entities meaning that they should hold properties and if required, some custom constructors. Utils methods can be moved to either a new service/services or to a utils class.